notebooks/data-sources/oc-github-repo/github_PR_EDA.ipynb

```diff
@@ -283,7 +283,7 @@
     "\n",
```

Comment:
<p> "points are centered around (0,0) in all 3 cases" --&gt;  "points are centered around (0,0) for the 3 pairs of features in the top row"</p><p> </p><p>Also can you make this a single paragraph. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/289/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>